src/Core/Framework/DataAbstractionLayer/ExtensionRegistry.php

```diff
@@ -11,6 +11,8 @@
 /**
  * @internal
  *
+ * @deprecated tag:v6.7.0 - unused with 6.7, will be now handled by \Shopware\Core\System\DependencyInjection\CompilerPass\SalesChannelEntityCompilerPass - reason:remove-entity
```

Comment:
The class is already internal, any reason we don't remove it directly?

Reply:
I would like to enable it only for 6.7 to see if I fucked up something while RC phase